scripts/world_bank/output/WorldBank.tmcf

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
nit: nice to remove extra first line

Reply:
Done, but shited the extra line to the end. I'm not sure that's any better.